DuckDuckGo/MainViewController+KeyCommands.swift

```diff
@@ -46,11 +46,33 @@ extension MainViewController {
                 UIKeyCommand(input: UIKeyCommand.inputLeftArrow, modifierFlags: [.command], action: #selector(keyboardBrowserBack),
                              discoverabilityTitle: UserText.keyCommandBrowserBack),
                 UIKeyCommand(input: "w", modifierFlags: .command, action: #selector(keyboardCloseTab),
```

Comment:
I'd like to be able to close the home  tab with `cmd+w` - what do you think @brindy?

Reply:
Yep will change that